package.json

```diff
@@ -19,7 +19,7 @@
     "@mui/x-data-grid": "^5.4.0",
     "@reduxjs/toolkit": "^1.7.1",
     "@sentry/nextjs": "^6.17.4",
-    "@superfluid-finance/sdk-core": "0.3.3-dev.4c31a63.0",
+    "@superfluid-finance/sdk-core": "0.4.2-dev.b0d8dd0.0",
     "@superfluid-finance/sdk-redux": "0.3.1-dev.812e77d.0",
```

Comment:
I guess we are using unreleased feature still?

Reply:
We haven't done a stable release after the big refactoring. Console and the Dashboard V2 have been the grounds for testing it. It has held up well and I don't mind doing a stable release. Would you be so kind and do it? The changelog is already there.

Reply:
you have the permission of managing the `origin/release-sdk-redux-stable` branch, Vincent already has the experience of release branches you can ask him about it, in short:

- if there is no divergence between release branch "dev" branches in case of hotfixes: then bump the package version of sdk-redux in "dev" -> sync release branch with the "dev" -> push -> and a release draft will be created in github releases.
- if there is divergence between release branch and "dev" branches due to hotfixes and not wanting to pull all the changes from "dev" yet, then only more "hot fixes" will go to the release branches
   - at some point, the release branch should be "force-synced" with "dev" branch to release a new release (usually a minor version release)